consensus/polybft/checkpoint_manager_test.go

```diff
@@ -593,7 +572,7 @@ func deployRootchainContract(t *testing.T, transition *state.Transition, rootcha
 		contracts.BLSContract,
 		bn256Addr,
 		bls.GetDomain(),
-		accSet.AsGenericMaps()})
+		accSet.ToAPIBinding()})
```

Comment:
Can this `initialize` method be encoded too?

Reply:
Done: https://github.com/0xPolygon/polygon-edge/pull/1166/commits/c97b2dc7208e1ff0800cdb1c99577e987f9bdb51.
Had to do two things:
1. Most of the contracts have `initialize` function (similar to `init` in `go`), so I had to make them unique somehow.
2. Some functions in the same contract use the same struct type as one of their parameters, so, to not generate the same structs 'n' times, had to keep count on which structs we generated already.